patches/tModLoader/Terraria/release_extras/tMLMod.targets

```diff
@@ -39,7 +39,12 @@
 	</ItemGroup>
 
 	<Target Name="BuildMod" AfterTargets="Build">
-		<Exec Command="dotnet $(tMLServerPath) -build $(ProjectDir) -eac $(TargetPath) -define &quot;$(DefineConstants)&quot; -unsafe $(AllowUnsafeBlocks) $(ExtraBuildModFlags)" WorkingDirectory="$(tMLSteamPath)"/>
+		<Exec Command="dotnet $(tMLServerPath) -build $(ProjectDir) -eac $(TargetPath) -define &quot;$(DefineConstants)&quot; -unsafe $(AllowUnsafeBlocks) $(ExtraBuildModFlags)" WorkingDirectory="$(tMLSteamPath)" ConsoleToMSBuild="true" ContinueOnError="ErrorAndContinue">
+			<Output TaskParameter="ConsoleOutput" PropertyName="ModBuildOutput"/>
```

Comment:
I don't see a use of `ModBuildOutput`. Do output and errors still show in the normal build window?

Reply:
Right, I was testing earlier to output the console output to the error instead and it was more confusing. I'll remove that. We can do string operations on the output to display a portion of the output one day if we ever need to.